src/convertToObject.js

```diff
@@ -7,6 +7,14 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  return sourceString
+    .split(';')
+    .map((keyVal) => keyVal.split(':').map((str) => str.trim()))
+    .reduce((accumulator, [key, value]) => {
+      accumulator[key] = value;
+
+      return accumulator;
+    }, {});
 }
```

Comment:
The implementation of the `convertToObject` function is correct and follows good practices like using `split`, `map`, and `reduce` to transform the string into an object. However, it's important to note that the function does not handle cases where there are multiple semicolons without any styles between them, which could lead to empty keys being added to the object. While handling this edge case is not required for this task, it would be a nice addition to check for and skip empty keys.